site/earlgrey-diagram/src/metadata.js

```diff
@@ -0,0 +1,239 @@
+export default {
+    "opentitan-logo": {
+        title: "opentitan",
+        href: "/",
+    },
+    "high-speed-crossbar": {
+        title: "high-speed-crossbar",
+        metrics: "tlul",
+        href: "/hw/ip/tlul#top",
+        report: "/hw/top_earlgrey/ip/xbar_main/dv/autogen",
+    },
+    "ibex": {
+        title: "ibex",
+        metrics: "rv_core_ibex",
+        href: "/hw/ip/rv_core_ibex#top",
+        report: "TODO",
```

Comment:
When leaving TODO comments in code that gets merged, our [guideline](https://github.com/lowRISC/opentitan/blob/master/doc/contributing/detailed_contribution_guide/README.md#what-about-todo-comments) is to create an issue and mention its number with the TODO text (e.g., `TODO(#12345)`). Does that make sense here as well?

Reply:
I would prefer setting it to `null` as well